### PR TITLE
Updates to UG

### PR DESCRIPTION
**Pull request by tsy24**
Add anchor links in section 4

Add additional information for `addTag`, `deleteTag`, `filter`, `undo`, `redo`

### CODE REVIEW

**Review by nicole-luo-exe — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -291,7 +290,7 @@ Examples:
 * The order of the keywords does not matter. e.g. `Elena Kro` will match `Kro Elena`
 * Only the name is searched.
 * Only full words will be matched. e.g. `Ele` will not match `Elena`
-* Persons matching at least one keyword will be returned (i.e. `OR` search). e.g. `Elena Kro` will return `Elena Grob`, `Kro Stanly`
+* Elderlies matching at least one keyword will be returned (i.e. `OR` search). e.g. `Elena Kro` will return `Elena Grob`, `Kro Stanly`
```
Comment (xiongjya) [inline docs/UserGuide.md:293]: Can you add a full stop after this sentence?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -627,24 +636,38 @@ Undoes the previous undoable command executed on the NurseyBook.
...
-* If there are no undoable commands executed previously, the undo command will fail and an error message will be shown.
-
 Example:
 * `deleteElderly 1` followed by `undo` causes the `deleteElderly 1` command to be undone and no elderly is deleted from the NurseyBook.
 
+<div markdown="block" class="alert alert-info">
+
+:information_source: **Information:**
+
+* Undoable commands (i.e. any command that modifies NurseyBook's data): `addElderly`, `editElderly`, `deleteElderly`, `deleteNok`, `addTag`, `deleteTag`, `addTask`, `editTask`, `deleteTask`, `doneTask`, `clear`
```
Comment (xiongjya) [inline docs/UserGuide.md:646]: Can you add the `remark` command under here?

**Review by xiongjya — CHANGES_REQUESTED** [review]
Looks good! When making the changes requested, can you run a check to ensure that all of the sentences that should have full stops do end with a full stop?

**Review by xiongjya — APPROVED** [review]
LGTM! 👍

**Pull request merged**